do/serverless.go

```diff
@@ -204,6 +230,7 @@ type serverlessService struct {
 	credsDir      string
 	node          string
 	userAgent     string
+	accessToken   string
```

Comment:
We are still using the plugin for 'doctl sls deploy|undeploy' and it cannot contact the triggers API without an actual access token.